app/controllers/users/notices_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Users::NoticesController < ApplicationController
+  def create
+    reciever = User.find(params[:reciever_id])
```

Comment:
__[Misspell]__

"reciever" is a misspelling of "receiver"


<!-- Comment by [Sider](https://sider.review) -->